content/docs/user-guide/packaging/windows-release-builds.md

```diff
@@ -1,296 +1,425 @@
+3. Click **Browse...**, which opens the File Explorer.
 
-4. Enter a name for your package file and save. In this example, we'll use the name `game_bundle.pak`.
+4. In File Explorer, enter a name for your package file, and then click **Save**. For this example, use the name `game.pak`.
 
 
 ### Create a bundle for engine assets
 
-Now, you'll create a bundle for your project's engine assets. 
+Next, create a bundle for your project's engine assets.
 
 #### Generate an asset list from default seed lists
 
-1. Navigate to the Seeds tab. 
+1. In the Asset Bundler, navigate to the **Seeds** tab.
 
-2. Select **Default Seed Lists** at the bottom of the **Seed List files** panel. Make sure to deselect other seed list files. 
+2. At the bottom of the **Seed List files** panel, select the **Default Seed Lists** check box. Make sure to deselect the `GameSeedList` file.
 
-2. Click **Generate Asset Lists**. This will open the **Generate Asset List files** dialog. 
+3. Click **Generate Asset Lists**, which opens the **Generate Asset List files** dialog.
 
-3. In the list of platforms, select **pc** by enabling the check box.  
+4. In the list of platforms, select the **pc** check box.
 
-4. Click **Browse...** to open the File Explorer. 
+5. Click **Browse...**, which opens File Explorer.
 
-5. Enter a name for your asset list and save. In this example, we'll use the name `engine.assetlist`. 
+6. In File Explorer, enter a name for your asset list, and then click **Save**. For this example, we'll use the name `engine.assetlist`.
 
-5. Navigate to the **Asset Lists** tab to verify the assets in `engine.assetlist`. The assets are listed under the **Asset List** panel. 
+7. In the Asset Bundler, navigate to the **Asset Lists** tab to verify the assets in `engine.assetlist`. The assets are listed in the **Asset List** panel.
 
 #### Bundle your assets
 
-1. Select your `engine.assetlist` in the** Asset List Files** panel under the **Asset Lists** tab.
+1. In the Asset Bundler, on the **Asset Lists** tab, in the **Asset List Files** panel, select the `engine.assetlist` file.
+
+2. Click **Generate Bundle**, which opens the **Generate Bundles** dialog.
+
+3. Click **Browse...**, which opens File Explorer.
+
+4. In File Explorer, enter a name for your package file, and then click **Save**. For this example, use the name `engine.pak`.
+
+
+For Windows, when Asset Bundler saves your bundle, it appends `_pc` to the bundle's name. So, you now have two asset bundles: `game_pc.pak` and `engine_pc.pak`.
+
 
-2. Click **Generate Bundle**. This opens the **Generate Bundles** dialog. 
+### Add bundles to the project game release layout
 
-3. Click **Browse...** to open the File Explorer. 
 
-4. Enter a name for your package file and save. In this example, we'll use the name `engine_bundle.pak`.
+Next, add your `game_pc.pak` and `engine_pc.pak` files to your project game release layout, so the Game Launcher can load assets from those bundles.
```

Comment:
```suggestion
Next, add your `game_pc.pak` and `engine_pc.pak` files to your project game release layout, so that the Game Launcher can load assets from those bundles.
```